biome.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.6.4/schema.json",
+	"files": {
+		"include": ["**/*.js"]
+	},
+	"organizeImports": {
+		"enabled": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	},
+	"javascript": {
+		"formatter": {
+			"quoteStyle": "single"
```

Comment:
FWIW, if you want to match our primary styleguide, you'd want a formatter config more like:

```json
{
  "formatter": {
    "indentStyle": "space",
    "indentWidth": 2,
    "lineWidth": 120
  },
  "javascript": {
    "formatter": {
      "quoteStyle": "single",
      "bracketSameLine": true,
      "bracketSpacing": false
    }
  }
}
```

Up to you if you want them to match or don't really mind either way (i'm all for just using defaults nowadays)

Reply:
Thanks!  I'm not heavily opposed to matching our style...  but to your point, I just really want defaults where it makes sense.  Except double quotes, I'll just never understand that as the default.